src/js/dex/component.js

```diff
@@ -18,6 +18,28 @@
             Currency.EUR,
             Currency.USD
         ];
+
+        ctrl.favoritePairs = [{
+            priceAsset: {abbr: 'WAV'},
```

Comment:
I'll also change it in the wavescore repository. Here, in DEX, it's just a placeholder for now.